update.ps1

```diff
@@ -1,50 +1,52 @@
 #####################################################
 # HelloID-Conn-Prov-Target-AFAS-Profit-Employees-Update
 #
-# Version: 2.1.0
+# Version: 2.1.0 | new-powershell-connector
```

Comment:
Do we need to do something about version number?
Also, how to differentiate between the versions? Powershell v1 vs v2 in HelloID I mean.